gnucash/CMakeLists.txt

```diff
@@ -44,6 +45,46 @@ TARGET_LINK_LIBRARIES (gnucash
    ${GUILE_LDFLAGS} ${GLIB2_LDFLAGS} ${GTK3_LDFLAGS} ${GTK_MAC_LDFLAGS}
 )
 
+# Get glib executable for generating the gresource file
+EXECUTE_PROCESS(
+   COMMAND
+       ${PKG_CONFIG_EXECUTABLE} gio-2.0 --variable glib_compile_resources
+   OUTPUT_VARIABLE
+       GLIB_COMPILE_RESOURCES_EXECUTABLE
+   OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+# Get the dependencies of the gresource
+EXECUTE_PROCESS(
+  OUTPUT_VARIABLE
+      gr_files
+  WORKING_DIRECTORY
+     ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
No need to set a working directory for this process. You work with full pathnames in the command below.

Reply:
Needed to leave this as the travis cmake ubuntu builds would fail

Reply:
Nope. The better solution would be to set --sourcedir to the command below rather than a WORKING_DIRECTORY. But you're right my suggestion did not work on ubuntu 14.04 either. Adding --sourcedir does work though.